.github/workflows/run-unit-tests.yml

```
@@ -0,0 +1,18 @@
+name: Manually Run Unit Tests
```

```suggestion
name: Run Unit Tests
```

This is an automatic process :)